utils.py

```diff
@@ -272,7 +272,12 @@ def convert_png_binary_to_data_url(content):
 
 
 def convert_png_to_data_url(filepath):
-    """Converts the png file at filepath to a data URL."""
+    """Converts the png file at filepath to a data URL.
+        Arg:
+            filepath: str type (path of the file)
```

Comment:
Can you please follow the style guide for writing docstrings provided [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#python)? You can also check the existing docstring to understand the pattern.